serverless.yml

```diff
@@ -130,7 +130,19 @@ stepFunctions:
               StartAt: tsDataTypeIdentifier
               States:
                 tsDataTypeIdentifier:
-                  Type: Pass
+                  Type: Task
+                  Resource: arn:aws:lambda:${self:provider.region}:${self:custom.accountNumber}:function:aqts-capture-stattype-router-${self:provider.stage}-determineRoute
+                  TimeoutSeconds: 60
+                  Retry:
+                    - ErrorEquals:
+                      - States.TaskFailed
+                      IntervalSeconds: 3
+                      MaxAttempts: 3
+                      BackoffRate: 2
+                  Catch:
+                    - ErrorEquals:
+                      - States.ALL
+                      Next: errorHandler
```

Comment:
I'm inclined to omit the `errorHandler` from the within map states and handle errors coming out of the map. A single failure from a map state causes the entire map to state to fail and all other runs within the map to abort. Rather than try to figure out which ones succeeded, failed, aborted, and never ran, it seem more reliable to have the `errorHandler` retry the entire map in the event of failure.

However, if we want to handle errors in the map states, an separate errorHandler (it could be the same Lambda function) needs to be defined within the context of the map state.